playbooks/roles/ocp-config/tasks/extract.yaml

```diff
@@ -31,6 +31,41 @@
   args:
     chdir: "{{ tools_dir }}"
 
+- name: Extract OCP4 tools from release image ( local-registry )
+  when: enable_local_registry
+  shell: |
+    oc adm release extract --tools {{ release_image_override }} --registry-config='{{ ansible_env.HOME }}/.openshift/pull-secret-updated'
+  args:
+    chdir: "{{ tools_dir }}"
+
+- name: Check if FIPS is enabled
+  ansible.builtin.command: grep -q 1 /proc/sys/crypto/fips_enabled
+  register: fips_check
+  changed_when: false
+  failed_when: false
```

Comment:
You could use the variable name `fips_compliant` which is available for you instead of this task.